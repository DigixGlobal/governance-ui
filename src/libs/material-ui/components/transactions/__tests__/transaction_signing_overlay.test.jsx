import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import TransactionSigningOverlay from '../transaction_signing_overlay';

jest.mock('react-redux', () => ({
  connect: () => component => component,
}));

describe('<TransactionSigningOverlay />', () => {
  test('renders correctly', () => {
    const component = shallow(
      <TransactionSigningOverlay
        hideTxSigningModal={jest.fn}
      />,
    );

    expect(toJson(component)).toMatchSnapshot();
  });
});
