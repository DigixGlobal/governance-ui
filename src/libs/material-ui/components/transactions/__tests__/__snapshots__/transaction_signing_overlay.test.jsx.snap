// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TransactionSigningOverlay /> renders correctly 1`] = `null`;
